local-demo/scripts/stop-demo.sh

```diff
@@ -29,5 +29,6 @@ ${SUDO} docker-compose -p dibichain-logistex -f docker-compose-local-logistex.ym
 ${SUDO} docker-compose -p dibichain-logistly -f docker-compose-local-logistly.yml down
 ${SUDO} docker-compose -p dibichain-operator -f docker-compose-local-operator.yml down
 ${SUDO} docker-compose -p dibichain-chain -f docker-compose-local-chain.yml down
+${SUDO} docker volume prune -f
```

Comment:
is it also possible to remove only the volumes which belongs to dibichain ?

Reply:
unfortunately not that easy, since there is no information of which volume belongs to which container if you do `docker volume ls`. But pruning only removes unused volumes. Do you think that's a problem? What could be the reason to have unused volumes dangling?